zio/parquetio/types.md

```diff
@@ -36,17 +36,27 @@ Un-annotated values of these types may be mapped to ZNG as follows:
 | BYTE_ARRAY   | `bstring` | The ZNG `bytes` type would be more appropriate if/when it is implemented |
 | FIXED_LEN_BYTE_ARRAY | `bstring` | (same as above) |
 
+Note: We have come across INT96 valued columns "in the wild" in files
```

Comment:
would you remove the text in the 'notes' column for int96, and just say 'see note below'